src/components/PostCard/PostCard.test.tsx

```diff
@@ -53,8 +53,8 @@ describe("Given the PostCard component", () => {
     });
   });
 
-  describe("When it receives a post with an alternative text: 'una araña'", () => {
-    test("Then it should show an alternative text 'una araña'", () => {
+  describe("When it receives a post with a image with an alternative text: 'una araña'", () => {
```

Comment:
When it receives a post with an image with the alternative text..

Reply:
cambiado